Pipfile

```diff
@@ -26,7 +26,7 @@ cchardet = "*"              # cchardet is a faster replacement for chardet
 aiofile = ">=2.0.0"         # used by util/async_file.py
 aiokafka = "*"              # pip install cryptofeed[kafka]
 hiredis = "*"               # pip install cryptofeed[redis]
-aioredis = ">=2.0.0a1"      # pip install cryptofeed[redis]
+aioredis = "*"      # pip install cryptofeed[redis]
```

Comment:
why did you change this? it will only work with that specific version of aioredis (the pre-release alpha). Older versions will not work.

Reply:
I see. On that version it was unable to resolve dependencies and create a lockfile via Pipfile.